File: app/src/main/java/com/waxym/defibapp/RetrofitInterface.java
package com.waxym.defibapp;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface RetrofitInterface {

    @GET("/api/records/1.0/search")
    Call<ResultModel> getResult(@Query("dataset") String dataset, @Query("rows") int rows);
}

File: app/src/main/java/com/waxym/defibapp/RetrofitInterface.kt
package com.waxym.defibapp

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface RetrofitInterface {

    @GET("/api/records/1.0/search")
    fun getResult(@Query("dataset") dataset: String, @Query("rows") rows: Int): Call<ResultModel>
}
